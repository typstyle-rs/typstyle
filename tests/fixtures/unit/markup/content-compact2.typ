#align(center, table(
  [
    ```
    underbrace(x + y, |A|)
    ```
  ],
))

#let list = (1,)
#[#while list.len() > 0 {
    list.pop()
  }\ ]
