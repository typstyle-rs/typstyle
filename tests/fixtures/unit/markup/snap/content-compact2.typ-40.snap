---
source: tests/src/unit.rs
input_file: tests/fixtures/unit/markup/content-compact2.typ
---
#align(center, table(
  [
    ```
    underbrace(x + y, |A|)
    ```
  ],
))

#let list = (1,)
#[#while list.len() > 0 {
  list.pop()
}\ ]
